docs/PRE-COMMIT.md

```diff
@@ -10,3 +10,13 @@ The pre-commit hook in this repository uses various plugins to run different kin
 1. [Pre-commit-hooks](https://github.com/pre-commit/pre-commit-hooks): Checks and fixes basic linting issues. All available hooks for pre-commit can be found [here](https://github.com/pre-commit/pre-commit-hooks#hooks-available).
 
 2. [Prettier mirror](https://github.com/pre-commit/mirrors-prettier): Checks for code formatting issues within Javascript files and fixes if errors found, like indentation, extra lines and spaces, invalid syntax, etc .
+
+#Installation
+Install pre-commit package manager.
+    pip install pre-commit
```

Comment:
Use code-blocks for better readability. Refer to `docs/INSTALLATION.md` for this.